pkgs/tools/filesystems/irods/icommands.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv, fetchFromGitHub, bzip2, zlib, avro-cpp, jansson, zeromq, cppzmq, openssl, kerberos, cmake, gnumake, llvmPackages, boost, libarchive, irods }:
+
+stdenv.mkDerivation rec {
+  version = "git-2016-10-24";
+  name = "irods-icommands-${version}";
+ 
+  buildInputs = [ bzip2 zlib cmake gnumake cppzmq libarchive avro-cpp jansson zeromq openssl llvmPackages.libcxx kerberos boost irods ];
+
+  propagateBuildInputs = [ boost ];
+
+  enableParallelBuilding = true;
+
+  src = [ ( fetchFromGitHub {
+            owner  = "irods";
+            repo   = "irods_client_icommands";
+            rev    = "179e8b5";
+            sha256 = "1gmv2frg0a3y5bdf21vz1cm5qccw5dp7gp7dcwwlf5qc9a9nixm7";
+          } )
+        ];
+
+  cmakeFlags = [
```

Comment:
A big chunk of this expression looks identical to default.nix in the same directory. Is there something sensible that could be factored out?